### PR TITLE
Migrate base_partner_merge to V10

### PR DESCRIPTION
**Pull request by alan196**

### CODE REVIEW

File: base_partner_merge/base_partner_merge.py [hunk 1]
```diff
@@ -1,11 +1,7 @@
 # -*- coding: utf-8 -*-
 # © 2016 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-from openerp.addons.crm.base_partner_merge import *  # noqa
-
-
-class NoCRMResPartner(ResPartner):  # noqa
-    _module = 'base_partner_merge'
+from openerp.addons.crm.wizard.base_partner_merge import *  # noqa
```
Comment (rousseldenis) [inline base_partner_merge/base_partner_merge.py]: @alan196 Since v10, use odoo.* in imports

File: base_partner_merge/validate_email.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf-8 -*-
 # © 2016 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-from openerp.addons.crm.validate_email import *  # noqa
+from openerp.addons.crm.wizard.validate_email import *  # noqa
```
Comment (rousseldenis) [inline base_partner_merge/validate_email.py]: @alan196 Same as above

File: base_partner_merge/base_partner_merge.py (hunk 1 above)
Comment (rousseldenis) [inline base_partner_merge/base_partner_merge.py]: @alan196 Put models in a 'models' directory
See https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#directories

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Review by ougc27 — APPROVED** [review]

**Comment by alan196** [comment]
@rousseldenis changes are done.

Thanks for your review 👍

**Comment by luistriana28** [comment]
+1

**Review by rousseldenis — APPROVED** [review]
:+1: Code review

File: base_partner_merge/views/base_partner_merge.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<openerp>
+<odoo>
     <data>
```
Comment (moylop260) [inline base_partner_merge/views/base_partner_merge.xml]: Remove unused `data`

**Review by moylop260 — APPROVED** [review]
👍 
(Just small change)

**Comment by alan196** [comment]
Change DONE Thanks @moylop260 for the review.

**Pull request merged**